channels/us.m3u

```diff
@@ -158,8 +158,6 @@ https://linear-12.frequency.stream/dist/roku/12/hls/master/playlist.m3u8
 https://linear-46.frequency.stream/dist/localnow/46/hls/master/playlist.m3u8
 #EXTINF:-1 tvg-id="AFVenEspanol.us" tvg-name="AFV en Español" tvg-country="US" tvg-language="Spanish" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5f7b66af901c6845181f33d3" group-title="Comedy",AFV en Español (720p)
 https://linear-46.frequency.stream/dist/roku/46/hls/master/playlist.m3u8
-#EXTINF:-1 tvg-id="AFVenEspanol.us" tvg-name="AFV en Español" tvg-country="US" tvg-language="Spanish" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/5f7b66af901c6845181f33d3" group-title="Comedy",AFV en Español (720p) [Offline]
-https://linear-46.frequency.stream/dist/plex/46/hls/master/playlist.m3u8
```

Comment:
Not 24/7

![image](https://user-images.githubusercontent.com/30985701/132143275-ce1c0ba9-26f5-4b1d-8bb7-d139eb3b3908.png)